static/src/stylesheets/layout/_header.scss

```diff
@@ -179,3 +147,648 @@ $c-guardian-services-action: #ffffff;
         float: right;
     }
 }
+
+
+/****************
+ * Mobile Header Styling below - TODO: Integrate when coming up with the desktop design
+ ****************/
+
+// Specific values for menu button breakpoints
+$gutter-small: 29px;
+$gutter-medium: 37px;
+$gutter-large: 55px;
+
+// When the menu is open this class is added to the html to prevent users from scrolling
+.nav-is-open {
+    overflow: hidden;
+    position: fixed;
+    width: 100%;
+}
+
+.new-header {
+    @include mobile-only;
+    background-color: $guardian-brand;
+}
+
+.new-header__inner {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    align-items: flex-start;
+}
+
+.new-header__logo-wrapper {
+    padding-top: $gs-baseline / 2;
+    padding-bottom: $gs-baseline / 2;
+    margin-left: auto;
+}
+
+.new-header__logo__svg {
+    // Removes extra spacing
+    display: block;
+    height: auto;
+    padding-right: $gs-gutter / 2;
+
+    // Aspect ratio: 16:3
+    @include mq($until: mobileMedium) {
+        width: 170px;
+        // height: auto doesn't work in Safari
+        height: calc(3 / 16 * 170px);
+    }
+
+    @include mq($from: mobileMedium, $until: mobileLandscape) {
+        width: 225px;
+        height: calc(3 / 16 * 225px);
+    }
+
+    @include mq(mobileLandscape) {
+        width: 260px;
+        // height: auto doesn't work in Safari
+        height: calc(3 / 16 * 260px);
+        padding: ($gs-baseline / 2) ($gs-gutter / 2) 0;
+        margin-right: $gs-gutter / 2;
+    }
+}
+
+.new-header__nav {
+    font-size: 0;
+    width: 100%;
+    overflow: hidden;
+
+    @include mq($until: mobile) {
+        // Menu squishes gracefully
+        height: $gs-baseline * 2.5;
+        margin-right: $gutter-small + $veggie-burger-small + $gs-gutter / 3;
+    }
+
+    @include mq(mobileLandscape) {
+        margin-left: $gs-gutter / 2;
+    }
+}
+
+.new-header__nav__menu-button {
+    // Override button from user agent stylesheet
+    position: absolute;
+    // Unset button from user agent stylesheet
+    border: 0;
+    outline: none;
+    // Override button from user agent stylesheet
```

Comment:
I don't understand those comments - don't they explain the whole block instead of single attributes?